atlp-rwanda/Team-Sostene-E-commerce-fe
# #185172110 Implement checkout functionality

#### What does this PR do?

- ensure that when a user can checkout after adding products to cart
- ensure that the user specifies the shipping address and gets a summary of the order
- ensure that the user is directed to the payment page after checking out

#### Description of Task to be completed?

- validating and processing the order
- redirecting the customer to the payment portal

#### How should this be manually tested?

- Login as a buyer
- on the view cart page, click proceed to checkout
- provided contact and shipping address info, then click confirm order.
- you will be redirected to payment page (Prince is working on the payment page).

#### Any background context you want to provide?

I will add a checkout button on the view cart page

#### What are the relevant pivotal tracker/Trello stories?

#185172110
